ibis/backends/duckdb/compiler.py

```diff
@@ -359,6 +359,10 @@ def visit_HexDigest(self, op, *, arg, how):
         else:
             raise NotImplementedError(f"No available hashing function for {how}")
 
+    @visit_node.register(ops.StringConcat)
+    def visit_StringConcat(self, op, *, arg):
+        return reduce(lambda x, y: sge.DPipe(this=x, expression=y), arg)
```

Comment:
Yes, these implementations are duplicated. No, I won't be refactoring that here 😄.